docs/core/deploying/index.md

```diff
@@ -1,95 +1,104 @@
 ---
```

Comment:
The switches are a little weird and change across different SDK versions. I didn't see a link in the body here to that.

Reply:
I'm not sure what exactly you're referencing. I have a switch table in the intro and these are the commands to produce those types of apps. .NET Core 2.2 introduced that kind of odd bridge between 2.1 and 3.0 but didn't have the defaults 3.0 has. Since 2.2 is out of support and not in this table, it's a lot easier to describe.